src/ticket-history/TicketHistoryScreenComponent.tsx

```diff
@@ -0,0 +1,86 @@
+import {ScreenHeading} from '@atb/components/heading';
+import {FullScreenView} from '@atb/components/screen-view';
+import {FareContractAndReservationsList} from '@atb/fare-contracts';
+import {ScrollContainer} from '@atb/fare-contracts/components/ScrollContainer';
+import {
+  FareContract,
+  filterExpiredFareContracts,
+  useTicketingState,
+} from '@atb/ticketing';
+import {useTimeContextState} from '@atb/time';
+import {
+  TicketingTexts,
+  TranslateFunction,
+  useTranslation,
+} from '@atb/translations';
+
+type Mode = 'expired' | 'sent';
+
+type Props = {mode: Mode};
+
+export const TicketHistoryScreenComponent = ({mode}: Props) => {
+  const {
+    fareContracts,
+    isRefreshingFareContracts,
+    rejectedReservations,
+    resubscribeFirestoreListeners,
+  } = useTicketingState();
+
+  const {serverNow} = useTimeContextState();
+
+  const {t} = useTranslation();
+
+  return (
+    <FullScreenView
+      headerProps={{
+        title: getTitle(mode, t),
+        leftButton: {type: 'back', withIcon: true},
+      }}
+      parallaxContent={(focusRef) => (
+        <ScreenHeading ref={focusRef} text={getTitle(mode, t)} />
+      )}
+    >
+      <ScrollContainer
```

Comment:
We end up with `ScrollView` inside `ScrollView` here, since `FullScreenView` already handles scrolling.

Reply:
removed!